templates/15_insert.tpl

```diff
@@ -83,13 +75,21 @@ func (o *{{$tableNameSingular}}) Insert(exec boil.Executor, whitelist ... string
 			{{end -}}
 		}
 
+		queryOutput := ""
```

Comment:
Tiny nit.

```go
var queryOutput, queryReturning string
```

I probably would have let this one go, but `queryReturning` is also mispelled so you'll have to touch this code regardless.